src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -16,13 +16,18 @@ public class HostMetrics : IHostMetrics
         private readonly ILogger _logger;
 
         public const string MeterName = "Microsoft.Azure.WebJobs.Script.Host.Internal";
+        public const string FaasMeterName = "Microsoft.Azure.Functions.Host";
         public const string CloudPlatformName = "azure_functions";
         public const string AppFailureCount = "azure.functions.app_failures";
         public const string ActiveInvocationCount = "azure.functions.active_invocations";
         public const string StartedInvocationCount = "azure.functions.started_invocations";
 
+        // FaaS Metrics
+        public const string FaasInvokeDuration = "faas.invoke_duration";
```

Comment:
Shouldnt this constant go with the others? 

[ResourceSemanticConventions.cs](https://github.com/Azure/azure-functions-host/pull/10929/files#diff-da26bd474f17401b95a7d3f6a925f1262665822cbccebea9a1d4f46341309b32) ?

Reply:
`faas.invoke_duration` is not a resource attribute. It is the specific to this meter.